sklearn/decomposition/pca.py

```diff
@@ -232,6 +235,9 @@ class PCA(_BasePCA):
         http://www.miketipping.com/papers/met-mppca.pdf. It is required to
         computed the estimated data covariance and score samples.
 
+        Equal to the average of (n_features - n_components)
```

Comment:
Rather than trying to describe a formula with words, maybe we could follow http://www.miketipping.com/papers/met-mppca.pdf and say something like: "this can be interpreted as the average variance ‘lost’ per discarded dimension"